app/parse.py

```diff
@@ -8,8 +18,53 @@ class Quote:
     tags: list[str]
 
 
+def get_one_quote(quote_tag: Tag) -> Quote:
+    return Quote(
+        text=quote_tag.select_one(".text").text,
+        author=quote_tag.select_one(".author").text,
+        tags=[tag.text for tag in quote_tag.select(".tag")]
+    )
+
+
+def get_all_quotes_from_page(page: Tag) -> list[Quote]:
+    page_quotes = page.select(".quote")
+
+    return [get_one_quote(quote) for quote in page_quotes]
+
+
+def get_all_quotes_from_site() -> list[Quote]:
+    logging.info("Start parsing!")
+    site = requests.get(SITE_URL).content
+    first_page = BeautifulSoup(site, "html.parser")
+
+    quotes = get_all_quotes_from_page(first_page)
+
+    i = 2
+    while True:
+        logging.info(f"Parse page №{i}")
+        site = requests.get(SITE_URL + f"page/{i}/").content
+        page = BeautifulSoup(site, "html.parser")
+
+        quotes.extend(get_all_quotes_from_page(page))
+        i += 1
+
+        if not page.select_one(".next"):
+            break
+
+    return quotes
+
+
+def write_quotes(output_scv_path: str, quotes: list[Quote]) -> None:
+    logging.info(f"Writing all quotes to {output_scv_path} file")
+    with open(output_scv_path, "w", newline="", encoding="utf-8") as file:
+        writer = csv.writer(file)
+        writer.writerow(["text", "author", "tags"])
+        for quote in quotes:
+            writer.writerow([quote.text, quote.author, quote.tags])
```

Comment:
The `tags` field is a list of strings, but it is being written directly to the CSV file. This will result in the list being converted to a string representation, which may not be the desired format. Consider joining the tags into a single string, separated by a delimiter (e.g., a comma or semicolon), before writing to the CSV.